sphinxcontrib/redoc.py

```diff
@@ -30,10 +32,24 @@ def render(app):
         # relies on them.
         ctx.setdefault('opts', {})
 
+        # "Embed" mode: the JSON spec is embedded in the page
```

Comment:
Can we change a comment into something like this? I think it might be helpful for readers to understand a rationale behind this chunk of code.

```python
        # In embed mode, we are going to embed the whole OpenAPI spec into
        # produced HTML. The rationale is very simple: we want to produce
        # browsable HTMLs ready to be used without any web server.
```

Reply:
yes, yours is better, i'll change it